Adding documentation for ELM questions v1 (#18356)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accesspackagelocalizedtext.md

---
title: "accessPackageLocalizedText resource type"

> description: "A complex type used to represent a string in a specific language."

Reviewer: ```suggestion
description: "Represents a question in a specific language."
```

author: "saylee-dharne-msft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessPackageLocalizedText resource type

Namespace: microsoft.graph

> A complex type used to represent a string in a specific language.

Reviewer: ```suggestion
Represents a question in a specific language.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |languageCode|String|The language code that `text` is in. For example, "en-us". The language component follows 2-letter codes as defined in [ISO 639-1](https://www.iso.org/iso-639-language-codes.html), and the country component follows 2-letter codes as defined in [ISO 3166-1 alpha-2](https://www.iso.org/iso-3166-country-codes.html). Required. |

Reviewer: ```suggestion
|languageCode|String|The language code that **text** is in. For example, "en-us". The language component follows 2-letter codes as defined in [ISO 639-1](https://www.iso.org/iso-639-language-codes.html), and the country component follows 2-letter codes as defined in [ISO 3166-1 alpha-2](https://www.iso.org/iso-3166-country-codes.html). Required. |
```

> |text|String|The text in the specific language. Required. |

Reviewer: ```suggestion
|text|String|The question in the specific language. Required. |
```
